.github/workflows/ci.yml

```diff
@@ -70,6 +78,9 @@ jobs:
     - name: Run brakeman
       run: bundle exec rake linter:brakeman
 
+    - name: Run openapifmt
```

Comment:
Just a brief note that when I was reviewing my initial thought on seeing this was: "it's good to do fmt in CI, but we probably out to lint as well". Only then realizing we actually already are. It makes me think that it might be good to change the name of the task though to be a bit more inclusive, ie `openapi_lint_and_fmt` or something. Or else something more generic like `openapici`. I don't know what exactly, but I do know calling it fmt made me jump to inaccurate conclusions.

Reply:
I think that's a too hard to type. But as @enescakir said, we already have something of an abuse of this notation and have the various code checkers...and formatters...in the `linter` namespace, so I guess I'll stick with the groove on this one (and move it in there, and call it `linter:openapi`)